static/stable/stage/services/services.json

```diff
@@ -30,37 +30,20 @@
         "isGroup": true,
         "title": "Red Hat Enterprise Linux",
         "links": [
-          "insights.remediations",
-          "insights.tasks"
+          "insights.remediations"
         ]
       },
       {
         "id": "ansible",
         "isGroup": true,
         "title": "Ansible",
         "links": [
-          {
```

Comment:
@johnsonm325 it looks like we are touching more than just the duplicate inventory and workspace links in this PR. I'll pull locally and make sure everything seems expected, but I'll be leaning on you to confirm the end results on stage.

Reply:
@johnsonm325 yes; those instructions should get you to the point where you have chrome-service-backend running locally, however in order to see/test things in the UI - you'll need to also run insights-chrome UI locally and point it to your running backend. You can do that by running the insights-chrome frontend with `CHROME_SERVICE=<chrome-service-backend-local-port> npm run dev` (see https://github.com/RedHatInsights/insights-chrome/blob/master/config/webpack.config.js#L86C23-L86C37)